Training25/Program.cs

```diff
@@ -4,10 +4,26 @@
 // ------------------------------------------------------------------
 // Program.cs
 // Program on main branch.
+// T6: Program to find the digital root of a number.
 // ------------------------------------------------------------------------------------------------
+using static System.Console;
 namespace Training25;
+
 internal class Program {
-   static void Main (string[] args) {
-      Console.WriteLine ("Hello, World!");
+   static void Main () {
+      do {
+         Clear ();
+         Write ("Enter any positive integer: ");
+         if (int.TryParse (ReadLine (), out int num) && num >= 0)
+            WriteLine ($"Digital root of {num} is {DigitalRoot (num)}");
+         else WriteLine ("Please enter a positive integer!");
+         Write ("Press 'Y' to continue");
+      } while (ReadKey ().Key == ConsoleKey.Y);

```

Comment:
```suggestion
      } while (ReadKey (true).Key == ConsoleKey.Y);
```
You can also pass 'true' to ReadKey () if you'd like to NOT display the entered value on the console